### DISCUSSION TITLE
New release: v0.23.0

### DISCUSSION THREAD

**Discussion by ObserverOfTime** [tree-sitter/py-tree-sitter#290] [Announcements]
This release changes the language function type.

- [README](https://github.com/tree-sitter/py-tree-sitter/blob/v0.23.0/README.md)
- [Release notes](https://github.com/tree-sitter/py-tree-sitter/releases/tag/v0.23.0)
- [API documentation](https://tree-sitter.github.io/py-tree-sitter/)

**Comment by nojaf** [comment]
Hello @ObserverOfTime, I'm encountering the [same problem](https://stackoverflow.com/questions/78949267/how-to-load-custom-language-in-python-tree-sitter-version-0-23-0) regarding loading a custom language.

I've built my language and would now like to use my "custom_language.dylib". How do we do this in v0.23?

**Reply by ObserverOfTime** [reply]
Use the generated python bindings of the language.

**Reply by nojaf** [reply]
Sorry, I'm a bit new to this, should https://tree-sitter.github.io/tree-sitter/cli/build.html have created a file I can load in Python?

**Reply by nojaf** [reply]
Okay, turns out my tree sitter project was a bit old and didn't have any bindings at all.
After running `tree-sitter init` and migrating some things it produced a `pyproject.toml` and I was able to use that in my project.
